.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[unstable]
+bindeps = true
```

Comment:
I don't really like us adding this file as it messes up local configs for devs. Perhaps we can commit it, and also add it to .gitignore so people can have their own local configs.

It makes it a little hard to specifically commit changes to the repo but I think it might be the best compromise.

Reply:
Yeah, that is unfortunate, thus I added this to the Known-issues-section. I put it in the project root for the CI to pick it up, as it is running the jobs from the project root. When running the command from `packages/engine`, the config will be merged as specified in [the cargo documentation](https://doc.rust-lang.org/cargo/reference/config.html). Thus, you could keep your file in `packages/engine/.cargo/config.toml` and it will be merged with the provided one.

I don't like to commit an ignored file, we could simply not commit it at all and mention it in the docs, that the flag has to be passed or the config has to be generated.